src/commands/InitCommand.js

```diff
@@ -84,7 +84,7 @@ export default class InitCommand extends Command {
     const versionLocation = this.repository.versionLocation;
     if (FileSystemUtilities.existsSync(versionLocation)) {
       this.logger.info("Removing old VERSION file.");
-      FileSystemUtilities.unlinkSync(versionLocation, "0.0.0");
+      FileSystemUtilities.unlinkSync(versionLocation);
```

Comment:
i don't see any version of `unlinkSync` in the history having a second argument. not sure why it's here.